.github/workflows/python-tests.yml

```diff
@@ -9,6 +9,7 @@ on:
 
 env:
   CARGO_TERM_COLOR: always
+  CI: true
```

Comment:
Isn't there a special variable set by github actions?
We previously had a condition with `os.environ.get("CI")` in some tests and  I think they were working, however, we didn't have it specified here